resources/standards-data/ldml-keyboards/techpreview/fetch-latest-cldr-techpreview.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# Copyright:    © SIL International.
+# Description:  Import new files from CLDR tech preview
+# Create Date:  17 Oct 2022
+# Authors:      Steven R. Loomis (SRL)
+
```

Comment:
```suggestion

set -eu

```

We try to use both of these for all scripts (exit on error, error on undefined variable). Reduces mistakes.